src/run_cmd.c

```diff
@@ -139,6 +139,7 @@ void		run_cmd(t_mshell *mshell, char *cmd)
 	free(cmd);
 	while (i < cmd_count)
 	{
+		replace_env(mshell, cmds, i);
```

Comment:
Waarom zou je hier de hele array meegeven? Kan je niet gewoon `cmds[i]` geven?

Reply:
yap, stuk beter, is aangepast.